src/docs/topics/docker/README.md

```diff
@@ -0,0 +1,168 @@
+!./src/*
+
+# Ignore any App_Data folder
+**/App_Data/
+
+# Ignore all prebuild
+**/[b|B]in/
+**/[O|o]bj/
+```
+
+## Docker
+
+First example is a simple one. Use Docker to build an image and run it (inside a container). 
+
+```cmd
+REM Folder where the Dockerfile stands
+cd /orchardcore
+
+REM Build image from Dockerfile
+docker build -t oc .
+
+REM Creates a container, runs it and expose it's service on port 80
+docker run -p 80:80 oc
+```
+
+## Prune intermediary images
+
+### When using `docker` command : 
+
+```cmd
+REM Prunes intermediary containers created while building by using --rm
+docker build -t oc --rm .
+
+REM Prunes all intermediary images
+docker image prune -f --filter label=stage=build-env
+
+REM Creates a container, runs it and expose it's service on port 80
+docker run -p 80:80 oc
+```
+
+Using these commands should get you a fully functional docker container running on port 80 so that you can access it with your browser by simply going to http://localhost. Though we assume that this will only allow you to use SQLite. For avoiding needing to install anything directly on your Docker host computer and getting everything running quickly you should use `docker-compose`.
+
+## Docker compose
+
+Docker Compose will allow you to deploy everything by doing simply `docker-compose up` command in the root folder of Orchard Core source code. Of course it requires that you have a docker-compose.yml file standing in that folder first. In the example shown below, we will create services for each of the database providers Orchard Core "officially supports".
+
+docker-compose.yml file example :  
+
+```YML
+version: '3.3'
+services:
+    web:
+        build: 
+            context: .
+            dockerfile: Dockerfile
+        ports:
+            - "5009:80"
+        depends_on:
+            - sqlserver
+            - mysql
+            - postgresql
+    sqlserver:
+        image: "mcr.microsoft.com/mssql/server"
+        environment:
+            SA_PASSWORD: "P@ssw0rd!123456"
+            ACCEPT_EULA: "Y"
+    mysql:
+        image: mysql:latest
+        restart: always
+        environment:
+            MYSQL_DATABASE: 'orchardcore_database'
+            MYSQL_USER: 'orchardcore_user'
+            MYSQL_PASSWORD: 'orchardcore_password'
+            MYSQL_ROOT_PASSWORD: 'root_password'
+        ports:
+            - '3306:3306'
+        expose:
+            - '3306'
+        volumes:
+            - mysql-data:/var/lib/mysql
+    postgresql:
+        image: postgres:latest
+        volumes:
+            - postgresql-data:/var/lib/postgresql/data
+        ports:
+            - 5432:5432
+        environment:
+            POSTGRES_USER: orchardcore_user
+            POSTGRES_PASSWORD: orchardcore_password
+            POSTGRES_DB: orchardcore_database
+volumes:
+    mysql-data:
+    postgresql-data:
+
+```
+## Prune intermediary images
+
+### When using `docker-compose` command : 
+
+```cmd
+REM Build images if they are not already
```

Comment:
if they are not already => if needed

Reply:
Builds images if they are not already built

I want to point on the fact that they could not be built at this point. Of course they are needed.
See distinction between `docker-compose build` and `docker-compose up`

I think it is fine or it needs to be rephrased. But "if needed" seems vague.